src/native/eventpipe/ds-dump-protocol.h

```diff
@@ -61,5 +61,27 @@ ds_dump_protocol_helper_handle_ipc_message (
 	DiagnosticsIpcMessage *message,
 	DiagnosticsIpcStream *stream);
 
+/*
+* DiagnosticsGenerateCoreDumpResponsePayload
+*/
+
+#if defined(DS_INLINE_GETTER_SETTER) || defined(DS_IMPL_DUMP_PROTOCOL_GETTER_SETTER)
+struct _DiagnosticsGenerateCoreDumpResponsePayload {
+#else
+struct _DiagnosticsGenerateCoreDumpResponsePayload_Internal {
+#endif
+	// uint = 4 little endian bytes
+	// string = (array<char> where the last char must = 0) or (length = 0)
+
+	int32_t error;
+	const ep_char16_t *error_message;
```

Comment:
If the struct is the owner of the string, it should be ep_char16_t *, if not it could be const ep_char16_t, but then you should pass its value into init function. Since you only access these fields from ds-dump-protocol.c it is ok to not use getter and setters for these fields.